statsmodels/genmod/generalized_linear_model.py

```diff
@@ -877,6 +878,21 @@ def fit(self, start_params=None, maxiter=100, method='IRLS', tol=1e-8,
             values for gradient optimization.  Only relevant if
             `method` is set to something other than 'IRLS'.
 
+        If IRLS fitting used, the following additional parameters are
+        available:
+
+        atol : float, optional
+            The absolute tolerance criterion that must be satisfied. Defaults
+            to ``tol``. Convergence is attained when:
+            :math:`rtol * prior + atol > abs(current - prior)`
+        rtol : float, optional
+            The relative tolerance criterion that must be satisfied. Defaults
+            to 0 which means ``rtol`` is not used. Convergence is attained
+            when:
+            :math:`rtol * prior + atol > abs(current - prior)`
+        tol_criterion : str, optional
+            Defaults to ``'deviance'``. Can optionally be ``'params'``.
+
         Notes
         -----
         This method does not take any extra undocumented ``kwargs``.
```

Comment:
can you delete this Notes section, 
AFAICS, this is outdated because there are also optimization kwargs for fit_gradient